attributes/default.rb

```diff
@@ -17,7 +17,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-default['telegraf']['config_file_path'] = '/etc/opt/telegraf/telegraf.conf'
+# version of telegraf to install, e.g. '0.10.0-1', '0.2.4-1' or nil for the latest
+default['telegraf']['version'] = '0.10.0-1'
+
+if node['telegraf']['version'].start_with? '0.2.'
+  default['telegraf']['config_file_path'] = '/etc/opt/telegraf/telegraf.conf'
+else
+  default['telegraf']['config_file_path'] = '/etc/telegraf/telegraf.conf'
+end
 default['telegraf']['config'] = {
```

Comment:
The whitespace/newlines is out of whack in this file (not all from this PR), can you clean that up?  I prefer no newline between default attributes but you can add newlines around the conditionals.  Thanks!